Bottom: 3b926e44a71661ef286acffbb1df022746d14797
Top:    1fbcd6680329505a3aba0e4b7a72ef1579e16102
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-29 17:46:41 +0100

Refresh of play-with-glsl-shader-for-rend

---

diff --git a/src/hid/common/hidgl.c b/src/hid/common/hidgl.c
index 6932388..0649fac 100644
--- a/src/hid/common/hidgl.c
+++ b/src/hid/common/hidgl.c
@@ -870,26 +870,6 @@ hidgl_finish_render (void)
   in_context = false;
 }
 
-void
-hidgl_start_render (void)
-{
-  if (in_context)
-    fprintf (stderr, "hidgl: hidgl_start_render() - Already in rendering context!\n");
-
-  in_context = true;
-  hidgl_init ();
-  hidgl_init_triangle_array (&buffer);
-}
-
-void
-hidgl_finish_render (void)
-{
-  if (!in_context)
-    fprintf (stderr, "hidgl: hidgl_finish_render() - Not currently in rendering context!\n");
-
-  in_context = false;
-}
-
 int
 hidgl_stencil_bits (void)
 {
